Sketchfab-Exporter.pyp

```diff
@@ -182,16 +198,32 @@ class PublishModelThread(threading.Thread):
 
         # Need to work on this some more
         time_start = datetime.datetime.now()
-        #t = time_start.strftime("%a %b %d %I:%M %p")
+        # t = time_start.strftime("%a %b %d %I:%M %p")
         t = time_start.strftime("%c")
 
         print("\nUpload started on {0}".format(t))
         print("Exporting...\n")
 
-        exportFile = os.path.join(self.activeDocPath, self.title + '.dae')
+        exportFile = os.path.join(self.activeDocPath, self.title + '.fbx')
+
+        options = self.get_fbxexport_options()
+        backup_options = {}
 
-        # COLLADA 1.4
-        documents.SaveDocument(self.activeDoc, exportFile, c4d.SAVEDOCUMENTFLAGS_DONTADDTORECENTLIST, COLLADA14)
+        EXPORT_OPTIONS[c4d.FBXEXPORT_TRACKS] = self.enable_animation
+        EXPORT_OPTIONS[c4d.FBXEXPORT_BAKE_ALL_FRAMES] = self.enable_animation
```

Comment:
usually, uppercase variables are read only variables. Could you try to change the loop beneath maybe?